### PR TITLE
Stricter types for Phoenix Channel status values

### PR DESCRIPTION
**Pull request by aloukissas**

### CODE REVIEW

File: types/phoenix/index.d.ts [hunk 1]
```diff
@@ -6,13 +6,15 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.4
 
+export type ChannelStatus = 'ok' | 'error' | 'timeout';
```
Comment (princemaple) [inline types/phoenix/index.d.ts:9]: PushStatus?

**Review by princemaple — COMMENTED** [review]
Just a naming suggestion

**Review by princemaple — APPROVED** [review]
LGTM

**Review by John-Goff — APPROVED** [review]

**Comment by aloukissas** [comment]
I don't have merge rights - what's the process of merging this with upstream?

**Comment by princemaple** [comment]
A DefinitelyTyped maintainer needs to do a final approval. We just wait.

**Review by rbuckton — APPROVED** [review]

**Comment by princemaple** [comment]
Ready to merge

**Pull request merged**